frontend/src/components/tasks/createTask.jsx

```diff
@@ -11,10 +11,7 @@ let CreateTask = React.createClass({
   propTypes: {
     showModal: React.PropTypes.bool.isRequired,
```

Comment:
can we add `project: React.PropTypes.number.isRequired`? It is needed for this component to create tasks, and this way React will throw error if it is the wrong type or is not provided.